WooCommerce/Classes/ViewRelated/Survey/SurveySubmittedViewController.swift

```diff
@@ -1,8 +1,20 @@
 import UIKit
 
+/// Outputs of the the `SurveySubmittedViewController`
+///
+protocol SurveySubmittedViewControllerOutputs: UIViewController {
```

Comment:
Yeah, I struggled a bit with this name 🤪
I went for "outputs" because it represents who the VC communicate with the outside world. Actions could mean Inputs and outputs?